packages/extension-ui/src/Popup/Settings.tsx

```diff
@@ -3,34 +3,69 @@
 
 import type { ThemeProps } from '../types';
 
-import React from 'react';
+import React, { useCallback, useContext } from 'react';
 import styled from 'styled-components';
 
+import infoIcon from '../assets/information.svg';
+import trustedIcon from '../assets/trusted.svg';
+import { EditMenuCard, Svg } from '../components';
+import { ActionContext } from '../components/contexts';
 import useTranslation from '../hooks/useTranslation';
 import Header from '../partials/Header';
 
 interface Props extends ThemeProps {
   className?: string;
 }
 
-function CreateAccountMenu({ className }: Props): React.ReactElement<Props> {
+const SettingsMenuCard = styled(EditMenuCard)`
+margin-bottom: 16px;
+`;
+
+function Settings({ className }: Props): React.ReactElement<Props> {
   const { t } = useTranslation();
+  const onAction = useContext(ActionContext);
+
+  const goTo = useCallback((path: string) => () => onAction(path), [onAction]);
 
   return (
     <>
       <Header
+        text={t<string>('Settings')}
         withBackArrow
         withHelp
-        text={t<string>('Settings')}
       />
-      {/* TODO: */}
-      <div className={className}>PLACEHOLDER</div>
+      <div className={className}>
+        <SettingsMenuCard
+          description=''
```

Comment:
why empty? todo if empty for now please :)
there are a few places you use that empty description. If it's not used maybe you can remove this prop alltogether?